src/Composer/Installers/LavaLiteInstaller.php

```diff
@@ -4,7 +4,7 @@
 class LavaLiteInstaller extends BaseInstaller
 {
     protected $locations = array(
-        'package' => 'packages/{$name}/',
+        'package' => 'packages/{$vendor}/{$name}/',
```

Comment:
It is created a few days back and nobody started using it.
You can change it in confidence.